h/static/scripts/group-forms/components/ModerationStatusSelect.tsx

```diff
@@ -0,0 +1,60 @@
+import type { IconComponent } from '@hypothesis/frontend-shared';
+import { FilterIcon } from '@hypothesis/frontend-shared';
+import { CautionIcon } from '@hypothesis/frontend-shared';
+import { RestrictedIcon } from '@hypothesis/frontend-shared';
+import { CheckAllIcon } from '@hypothesis/frontend-shared';
+import { DottedCircleIcon, Select } from '@hypothesis/frontend-shared';
+
+export type ModerationStatus = 'pending' | 'approved' | 'denied' | 'spam';
+
+export type ModerationStatusSelectProps = {
+  selected?: ModerationStatus;
+  onChange: (status?: ModerationStatus) => void;
+};
+
+type Option = {
+  icon: IconComponent;
+  name: string;
+};
+
+const options = new Map<ModerationStatus, Option>([
+  ['pending', { name: 'Pending', icon: DottedCircleIcon }],
+  ['approved', { name: 'Approved', icon: CheckAllIcon }],
+  ['denied', { name: 'Denied', icon: RestrictedIcon }],
+  ['spam', { name: 'Spam', icon: CautionIcon }],
+]);
+
+export default function ModerationStatusSelect({
+  selected,
+  onChange,
+}: ModerationStatusSelectProps) {
+  const selectedName = selected ? options.get(selected)?.name : undefined;
+
+  return (
+    <Select
+      value={selected}
+      onChange={onChange}
+      alignListbox="right"
+      containerClasses="!w-auto"
+      aria-label="Moderation status"
+      buttonContent={
+        <div className="flex gap-x-1.5 items-center">
+          <FilterIcon />
+          <b>{selectedName ?? 'All'}</b>
```

Comment:
I think this looks better with a regular-weight font. This is also more consistent with eg. the "role" dropdown in the members tab.